addons-cluster/zookeeper/templates/cluster.yaml

```diff
@@ -14,20 +14,25 @@ spec:
       serviceAccountName: {{ include "kblib.serviceAccountName" . }}
       {{- include "kblib.componentMonitor" . | indent 6 }}
       {{- include "kblib.componentResources" . | indent 6 }}
+      {{- include "kblib.componentServices" . | indent 6 }}
```

Comment:
"kblib.componentServices" works with the legacy API (override service defined in `clusterDef.componentDefs[*].service`), the `clusterDef.componentDefs` fields have already been deleted, so it should not be included in main branch.

However, addons in 0.9 branch still have `clusterDef.componentDefs`.

Reply:
do not need to do any modification, @Y-Rookie will handle this situation uniformly.